src/index.html

```diff
@@ -5,10 +5,122 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>John Roberts' CV</title>
-    <link rel="stylesheet" href="./css/index.scss" />
-  </head>
-  <body>
+    <link rel="stylesheet" href="./css/index.scss"/>
+
+    <body>
     <div id="main"></div>
     <script src="./js/index.js"></script>
+      <nav> HOME </nav>
+      <button> Download my resume </button>
+
+      <!-- 主体第一部分 -->  
+      <h1>John Roberts's CV</h1>
+        <h2>BUSINESS EXPERT</h2>
+        <img src="smiley.gif" alt="Smiley face" height="42" width="42">
+
+      <!-- 主体第二部分 -->
+      <div>
+          <h3 class="double-colum">ABOUT ME</h3>
+              <p>I'm a paragraph. Click here to add your own text and edit me. It’s easy. Just click “Edit Text” 
+                or double click me to add your own content and make changes to the font. Feel free to drag and 
+                drop me anywhere you like on your page. I’m a great place for you to tell a story and let your 
+                users know a little more about you.</p>
+
+              <p>I'm a paragraph. Click here to add your own text and edit me. It’s easy. Just click “Edit Text” 
+                or double click me to add your own content and make changes to the font. Feel free to drag and 
+                drop me anywhere you like on your page. I’m a great place for you to tell a story and let your 
+                users know a little more about you.</p>
+
+          <h3>EXPERIENCE</h3>
+              <div>
+                  <b>Budget Director@Wix.com</b>
```

Comment:
might want to wrap a `p` tag around 'b'.
the `b` tag is not recommended to use since it has no semantical meanings.
you should use the CSS font-weight property to create boldface text, or the <strong> element to indicate that text is of special importance.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b